readthedocs/rtd_tests/tests/test_project_forms.py

```diff
@@ -1176,14 +1176,19 @@ def setUp(self):
     def test_addonsconfig_form(self):
         data = {
             "enabled": True,
+            "options_root_selector": "main",
             "analytics_enabled": False,
             "doc_diff_enabled": False,
+            "filetreediff_enabled": True,
+            "filetreediff_ignored_files": "user/index.html\nchangelog.html",
```

Comment:
Curious what happens if a user just does space delimited or similar?

Reply:
We consider one per line. I put that in the help text and also in the placeholder.

Reply:
I was more suggesting that we have tests for that behavior. It seems like we want to break it on spaces as well as newlines? Otherwise I assume users will struggle with this.

Reply:
Mostly I was asking for tests that show what happen when a user puts spaces in.